tests/ut_mcalendar/ut_mcalendar.cpp

```diff
@@ -875,6 +875,7 @@ void Ut_MCalendar::testIcuFormatString_data()
 
 void Ut_MCalendar::testIcuFormatString()
 {
+    QSKIP("Icu Format strings change between releases. Skipping this test.");
```

Comment:
_data function has #if !defined(ALSO_VERIFY_ICU_DOES_ITS_JOB_AS_WE_EXPECT)
which doesn't make sense anymore with skipping. Suggestion: remove that block and skip here based on such #if.